### PR TITLE
[Support] Fix the issue where the character being saved in Unicode causes a warning to be treated as an error in Visual Studio 2022.

### PR DESCRIPTION
**Pull request by harrisonGPU**
Fix the issue where the character being saved in Unicode causes a warning to be treated as an error in Visual Studio 2022.
image

### CODE REVIEW

File: llvm/lib/Support/Parallel.cpp [hunk 1]
```diff
@@ -57,7 +57,7 @@ class ThreadPoolExecutor : public Executor {
     Threads.resize(1);
     std::lock_guard<std::mutex> Lock(Mutex);
     // Use operator[] before creating the thread to avoid data race in .size()
-    // in “safe libc++” mode.
+    // in "safe libc++" mode.
```
Comment (jhuber6) [inline llvm/lib/Support/Parallel.cpp:60]:
```suggestion
    // in 'safe libc++' mode.
```
Single quotes are more common for stuff like this.

**Review by jhuber6 — COMMENTED** [review]
Can you add the relevant error message you're getting to the PR's description please.

**Comment by jhuber6** [comment]
Is this really all it takes for VSC to fail? It's just a comment, so I would assume it's not an issue.

**Comment by TSWorld1314** [comment]
image

**Review by jhuber6 — APPROVED** [review]

**Pull request merged**